pipenv/utils.py

```diff
@@ -496,7 +496,7 @@ def resolve_deps(
             except (ValueError, KeyError, ConnectionError, IndexError):
                 if verbose:
                     print('Error generating hash for {}'.format(name))
-            collected_hashes = list(set(collected_hashes))
+            collected_hashes = list(sorted(set(collected_hashes)))
```

Comment:
Is there ever a time when `sorted()` doesn't return a list? Can't we drop the `list()` call here?

Reply:
That's a good point.

While we're fixing this we might want to remove the other `sorted` calls that were added here https://github.com/pypa/pipenv/commit/4d8e01b80f1560ce0fc5535f2fc04bca8cf30927 since they don't end up having an effect on the final result.

Reply:
`sorted` is guarenteed to return a list, according to the docs. This might change in future Python releases (anything might, technically), but that would be highly unlikely.